### DISCUSSION TITLE
Dumping PEC in Python

### DISCUSSION THREAD

**Discussion by hiiamfrompoland** [thliebig/openEMS-Project#84] [Q&A Usage] [answered]
Hello,

I have freshly transferred to the Python interface and I would like to dump the PEC mesh to verify it. 
Previously it was done with an option --debug_PEC, I don't see how could I do it now?

**Comment by biergaizi** [comment, marked as answer]
Pass the parameter `debug_pec=True` to `Run()`, as in:

    from openEMS import openEMS
    FDTD = openEMS(NrTS=1e4, EndCriteria=1e-4)
    FDTD.Run(Sim_Path, debug_pec=True)

To learn more usages, check the source code of [openEMS.pyx](https://github.com/thliebig/openEMS/blob/master/python/openEMS/openEMS.pyx#L450) for details.

**Reply by hiiamfrompoland** [reply]
Thank you for the reply, it is it.
